doc/source/theory/dem/dem.rst

```diff
@@ -121,80 +124,122 @@ Where:
 
 * :math:`\nu_i, \nu_j` poisson coefficient of particle i or j;
 
-Rolling friction may be calculated through a constant torque model or a viscous torque model corresponding to those equations:
+~~~~~~~~~~~~~~~~~~~~
+Coulomb's limit
+~~~~~~~~~~~~~~~~~~~~
+
+Coulomb's criterion is breached when the following condition is broken during a collision:
 
 .. math::
-    \mathbf{M}_{ij}^{r} &= -\mu_{r}R_{e}|\mathbf{F}_{ij}^{n}| \mathbf{\hat{\omega}}_{ij} \\
-    \mathbf{M}_{ij}^{r} &= -\mu_{r}R_{e}|\mathbf{F}_{ij}^{n}||\mathbf{V}_{\omega}| \mathbf{\hat{\omega}}_{ij}
+    |\mathbf{F}_{ij}^{t}| \geq \mu |\mathbf{F}_{ij}^{n}|
 
-Where the parameters are:
+
+A breach means the collision is having gross sliding and tangential force needs to be limited to the Coulomb's limit.
+To do so, the tangential overlap :math:`\mathbf{\delta_t}` is first limited and then the tangential force is recalculated.
+
+The tangential overlap is computed from tangential force without damping force when using nonlinear viscoelastic contact model:
 
 .. math::
-    \mathbf{\hat{\omega}}_{ij} &= \frac{\omega_{i} - \omega_{j}}{|\omega_{i} - \omega_{j}|} \\
-    \mathbf{V}_{\omega} &= \left( \omega_{i} \times R_{i}\mathbf{n}_{ij}-\omega_{j} \times R_{j}\mathbf{n}_{ji} \right)
+    \mathbf{\delta_t} &= \frac{\mathbf{\tilde{F}_{ij}}}{-k_{t}} \\
+    \mathbf{\tilde{F}_{ij}} &= \mathbf{\hat{F}_{ij}} + \eta_{t}\mathbf{v}_{rt} \\
+    \mathbf{\hat{F}_{ij}^{t}} &= \mu |\mathbf{F}_{ij}^{n}| \frac{\mathbf{F}_{ij}^{t}}{|\mathbf{F}_{ij}^{t}|}
 
-Where:
+Regarding the particle-wall contacts, the models applied are the same as particle-particle contacts with a background triangulation and mapping with walls.
 
-* :math:`\mu_{r}` rolling friction coefficient;
+.. note::
+    When using a cohesive force models, Coulomb's criterion need to be modified. For further information, please read the sections associated with each cohesive force model.
 
-Tangential torque is calculated through:
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Tangential torque
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tangential torque is the torque generated by the tangential force. It can be calculated through:
 
 .. math::
-    \mathbf{M}_{ij}^{t} = R_{i}\mathbf{n}_{ij} \times \mathbf{F}_{ij}^{c}
+    \mathbf{M}_{ij}^{t} = R_{i}\mathbf{n}_{ij} \times \mathbf{F}_{ij}^{t}
+
+.. note::
+    As of now, the lethe-particles solver only uses spherical particles, thus the normal force does not generate a torque on the particle during a collision.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Rolling friction models
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Coulomb's criterion is violated when this condition is not respected during a collision:
+Rolling friction may be computed through a constant torque model or a viscous torque model. It can also be disabled using a no resistance torque model. The corresponding model can be described by the following equations:
```

Comment:
We are not talking about parameters on this page. Only theory. 

This would be inconsistent with the rest of this page. (ex. "young modulus particles" , "poisson ratio particles", etc.)

Also every contact force model.